### DISCUSSION TITLE
Permissive mode

### DISCUSSION THREAD

**Discussion by lsap** [Bogdan17-bit/Trading-Platform#4] [Q&A] [answered]
Should we enable aforementioned mode as w/o it there is an error while compiling
In file included from candles/candlegraphbuilder.cpp:1:
candles/candlegraphbuilder.h:25:22: error: declaration of ‘CandleStickList* CandleGraphBuilder::CandleStickList’ changes meaning of ‘CandleStickList’ [-fpermissive]
   25 |     CandleStickList *CandleStickList; // adapter
?

**Comment by Bohdan17-bit** [comment, marked as answer]
It's strange, even though it works for me, it's better to change the case of the first letter:
CandleStickList *candleStickList;

**Comment by Bohdan17-bit** [comment]
So, in the file tradewindow.cpp we must also rename
586 candle_graph->addAllCandleStickSets(candle_graph->CandleStickList->get_list_candlestick());
on "candleStickList".
